src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                +
+                "age="
+                + age
+                +
+                ", name='"
+                + name + '\''
+                +
+                ", ownerName='"
+                + ownerName + '\''
+                +
+                '}';
```

Comment:
It would be more readable if all variables will be in the same line with their values. Also consider providing `+ `operator before the beginning of each line
[Java Style Doc](https://mate-academy.github.io/style-guides/java/java.html#s4.5-line-wrapping)